src/function/scalar/compressed_materialization/compress_integral.cpp

```diff
@@ -228,6 +228,11 @@ ScalarFunction CMIntegralCompressFun::GetFunction(const LogicalType &input_type,
 	                      GetIntegralCompressFunctionInputSwitch(input_type, result_type), CMUtils::Bind);
 	result.serialize = CMIntegralSerialize;
 	result.deserialize = CMIntegralDeserialize<GetIntegralCompressFunctionInputSwitch>;
+#if defined(D_ASSERT_IS_ENABLED)
+	result.errors = FunctionErrors::CAN_THROW_RUNTIME_ERROR; // Can only throw runtime error when assertions are enabled
```

Comment:
Instead of patching it like this - maybe we can have the code that checks for this special case on internal exceptions (and let them through regularly)?

Internal exceptions shouldn't really be thrown as regular errors imo so I think setting this flag is unnecessary